Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -679,7 +679,22 @@ def setup_(self, step, stepName, stepDict, k, properties):
     offset = 0.512,
 )
 
-# add here a .513 workflow for GPU vs CPU validation
+upgradeWFs['PatatrackECALOnlyGPUValidation'] = PatatrackWorkflow(
+    digi = {
+        '--accelerators': 'gpu-nvidia',
+        '--procModifiers': 'gpu,gpuValidationEcal'
```

Comment:
`gpuValidationEcal` is not needed here, because I don't think we run any validation in the DIGI step

Reply:
Probably not.